packages/components/atoms/f-button/test/specs/component/f-button.component.spec.js

```diff
@@ -1,21 +1,32 @@
+const { buildUrl } = require('@justeat/f-wdio-utils/src/storybook-extensions.js');
 const Button = require('../../../test-utils/component-objects/f-button.component');
-const button = new Button();
+
+const button = new Button('atom-folder', 'f-button--button-component');
 
 describe('f-button component tests', () => {
 
     it('should display the f-button action component', () => {
+        // Arrange
```

Comment:
Are we changing line 10 to only pass in the component, like `buildUrl(button)`? I think you mentioned this before so may already be doing it 😄

Reply:
Correct but that requires changing the webdriver io utils function, which ill do as a different PR, then ill create a different PR to refactor the tests to just take in the 1 param.

Reply:
awesome, sounds good!